<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pakan</class>
 <widget class="QDialog" name="pakan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="namaPakanLabel">
      <property name="text">
       <string>Nama Pakan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namaPakanLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="jenisLabel">
      <property name="text">
       <string>Jenis</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="jenisLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jumlahPakanLabel">
      <property name="text">
       <string>Jumlah Pakan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="jumlahPakanLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
